server/routes.js

```diff
@@ -291,6 +292,7 @@ exports.isomorphicRoutes = function isomorphicRoutes(
     getClient = require("./api-client").getClient,
     renderServiceWorker = renderServiceWorkerFn,
     publisherConfig = require("./publisher-config"),
+    redirectUrls = [],
```

Comment:
How would ahead pass this? They have seperate url list for publishers.

Reply:
redirectUrls will have two values either it will be an object of an array or a function. taking array is the default value if nothing is passing here.